### PR TITLE
fix: better Invoker example

### PR DESCRIPTION
**Pull request by anna-carroll**
closes https://github.com/paradigmxyz/alphanet/issues/55

### CODE REVIEW

File: crates/testing/resources/eip3074/src/GasSponsorInvoker.sol [hunk 1]
```diff
@@ -18,13 +17,14 @@ contract GasSponsorInvoker is BaseAuth {
     /// @return success True if the call was successful
     function sponsorCall(
         address authority,
-        bytes32 commit,
         uint8 v,
         bytes32 r,
         bytes32 s,
         address to,
         bytes calldata data
     ) external pure returns (bool success) {
```
Comment (enitrat) [inline crates/testing/resources/eip3074/src/GasSponsorInvoker.sol:25]: I think the function should not be pure given that `authcall` can modify the state, right?

**Comment by fgimenez** [comment]
I've rebuilt the artifacts and pushed a smol change to use `SolValue::abi_encode_sequence` to generate the commit in the test and everything is working now, can you pls take a look @anna-carroll ?

**Review by fgimenez — APPROVED** [review]

**Comment by Rjected** [comment]
bump @anna-carroll for review

**Review by anna-carroll — COMMENTED** [review]
LGTM! sorry for the delay y'all

**Pull request merged**